### PR TITLE
API-6521 documented alternative_text for images in file event

### PR DESCRIPTION
**Pull request by olek-p**
## 🚀 Links

- [Feature branch](https://developers.labs.livechat.com/docs/feature/API-6521)
- [Jira](https://livechatinc.atlassian.net/browse/API-6521)

## 📓 Description

Support for `alternative_text` for images within File Events. Only available in v3.3 of agent-api and customer-api.

## ✅ Checklist (for API docs)

- Changelog
- API versions
- Web & RTM
- **Table:** Available methods/webhooks/pushes
- **Table:** Specifics

For more guidelines, see [Updating API docs](https://livechatinc.atlassian.net/wiki/spaces/PAT/pages/761529400/Updating+API+docs).

## 👷 Type of change (remove not needed)

New param in File Event

### Content

- New content
- Proofreading/content fixes

### CODE REVIEW

**Comment by olek-p** [comment]
@klarzynskik same thing here, but for images in File events

File: content/messaging/agent-chat-api/changelog/index.mdx [hunk 1]
```diff
@@ -9,6 +9,12 @@ desc: "Changelog of the Agent Chat API"
 
 This document is the record of all the changes in the **Agent Chat API**, Web and RTM references, starting from **version 3.0**.
 
+## [v3.3] - 2020-06-30 - Developer preview
+
+### Changed
+
+- There's a new parameter `alternative_text` for images within File Events ([Web](/messaging/agent-chat-api/v3.3/#file) & [RTM](/messaging/agent-chat-api/v3.3/rtm-reference/#file)).
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/changelog/index.mdx:16]:
```suggestion
- There's a new parameter `alternative_text` for images within the File event ([Web](/messaging/agent-chat-api/v3.3/#file) & [RTM](/messaging/agent-chat-api/v3.3/rtm-reference/#file)).
```

File: content/messaging/customer-chat-api/changelog/index.mdx [hunk 2]
```diff
@@ -9,6 +9,12 @@ desc: "Changelog of the Customer Chat API"
 
 This document is the record of all the changes in the **Customer Chat API**, Web and RTM, starting from **version 3.0**.
 
+## [v3.3] - 2020-06-30 - Developer preview
+
+### Changed
+
+- There's a new parameter `alternative_text` for images within File Events ([Web](/messaging/customer-chat-api/v3.3/#file) & [RTM](/messaging/customer-chat-api/v3.3/rtm-reference/#file)).
```
Comment (ale-kacperczyk) [inline content/messaging/customer-chat-api/changelog/index.mdx:16]:
```suggestion
- There's a new parameter `alternative_text` for images within the File event ([Web](/messaging/customer-chat-api/v3.3/#file) & [RTM](/messaging/customer-chat-api/v3.3/rtm-reference/#file)).
```

File: content/messaging/agent-chat-api/changelog/index.mdx [hunk 3]
```diff
@@ -9,6 +9,12 @@ desc: "Changelog of the Agent Chat API"
 
 This document is the record of all the changes in the **Agent Chat API**, Web and RTM references, starting from **version 3.0**.
 
+## [v3.3] - 2020-06-30 - Developer preview
+
+### Events
+
+- There's a new parameter `alternative_text` for images within the **File event** ([Web](/messaging/agent-chat-api/v3.3/#file) & [RTM](/messaging/agent-chat-api/v3.3/rtm-reference/#file)).
```
[the remark this answers is not in the document]
Comment (olek-p) [reply content/messaging/agent-chat-api/changelog/index.mdx]: Applied

File: content/messaging/agent-chat-api/changelog/index.mdx [hunk 4]
```diff
@@ -9,6 +9,12 @@ desc: "Changelog of the Agent Chat API"
 
 This document is the record of all the changes in the **Agent Chat API**, Web and RTM references, starting from **version 3.0**.
 
+## [v3.3] - 2020-06-30 - Developer preview
```
[the remark this answers is not in the document]
Comment (olek-p) [reply content/messaging/agent-chat-api/changelog/index.mdx]: Fixed

**Pull request merged**